recipe/conda-forge.yml

```diff
@@ -0,0 +1,4 @@
+azure:
```

Comment:
this needs to sit at the root of the feedstock

Reply:
pls check https://github.com/conda-forge/libignition-physics-feedstock/commit/c586d765a2f5fd0ecf6da43c53315c898c9bf6bd#diff-478786365dd93f740eb520c2faa03d7a0623273663b75472272c8ef94297bbe2

Reply:
Thanks! I moved it to root.